source/_integrations/asuswrt.markdown

```diff
@@ -19,7 +20,15 @@ There is currently support for the following device types within Home Assistant:
 - **Presence Detection** - The ASUSWRT platform offers presence detection by looking at connected devices to a ASUSWRT based router.
 - **Sensor** - The ASUSWRT sensor platform allows you to get upload and download data from your ASUSWRT within Home Assistant.
 
-## Configuration
+## Configuration via frontend
+
+To add your ASUSWRT devices into your Home Assistant installation, go to:
+
+**Configuration** -> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **ASUSWRT**.
+
+### Configuration via YAML
+
+_YAML configuration is still around for people that prefer YAML, but it's deprecated, and you should not use it anymore._
```

Comment:
YAML configuration should be removed from documentation when the configuration flow is introduced.